WordPress/gutenberg
# DataViews: make dataviews powered page patterns stable

Part of https://github.com/WordPress/gutenberg/issues/55083

## What?

This PR makes the dataviews-powered Patterns page stable.

## Why?

It's considered to have feature-parity with the existing.

## How?

- Removes the check for experiment in the router.
- Updates the contents of `page-patterns/index.js` with those of `page-patterns/dataviews-patterns.js` and removes the latter.

## Testing Instructions

Load the "Side Editor > Patterns" page with and without the "admin views" experiment enabled and verify that the dataviews-powered page is loaded.

<img width="1500" alt="Captura de ecrã 2024-01-23, às 18 08 49" src="https://github.com/WordPress/gutenberg/assets/583546/3e7153b6-1d82-4593-9fbe-0a7cd308ce5a">
